app/views/home/_header.html.erb

```diff
@@ -27,9 +27,8 @@
         <a class="nav-link disabled" href="#">Disabled</a>
       </li>
     </ul>
-    
-    <% if current_user.blank? %>
-      <%= button_to "Log In", user_companies_find_path, method: 'get', class: 'btn btn-outline-success my-2 my-sm-0 float-right'%>
+    <% if current_user.nil? %>
+      <%= button_to "Log In", user_companies_find_path, method: 'get', class: 'btn btn-outline-success my-2 my-sm-0 float-right mr-3'%>
       <%= button_to "Sign Up", new_user_registration_path, method: 'get', class: 'btn btn-outline-success my-2 my-sm-0 float-right'%>
```

Comment:
I think no need to specify method.new_user_registration_path will take care of it.same as at line 31